spk/rutorrent/src/service-setup.sh

```diff
@@ -27,14 +32,12 @@ PID_FILE="${SYNOPKG_PKGVAR}/rtorrent.pid"
 LOG_FILE="${SYNOPKG_PKGVAR}/rtorrent.log"
 SVC_WRITE_PID=y
 
+SERVICE_COMMAND="env RUTORRENT_WEB_DIR=${RUTORRENT_WEB_DIR} SYNOPKG_PKGVAR=${SYNOPKG_PKGVAR} SYNOPKG_PKGDEST=${SYNOPKG_PKGDEST} ${SERVICE_COMMAND}"
+
 validate_preinst ()
 {
     if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
-        if [ ! -d "${wizard_download_dir}" ]; then
-            echo "Download directory ${wizard_download_dir} does not exist."
-            exit 1
-        fi
-        if [ -n "${wizard_watch_dir}" -a ! -d "${wizard_watch_dir}" ]; then
+        if [ -n "${wizard_watch_dir}" ] && [ ! -d "${wizard_watch_dir}" ]; then
```

Comment:
FYI, I've migrated deluge's watch directory to be `watch-deluge` so rutorrent and deluge do not collide.

Reply:
we should keep the sh/ash compatible variant `if [ -n "${wizard_watch_dir}" -a ! -d "${wizard_watch_dir}" ]; then` and not use the bash syntax (&&) here (correct me if && is supported in ash).